pkg/volume/plugins.go

```diff
@@ -199,6 +199,13 @@ type AttachableVolumePlugin interface {
 	GetDeviceMountRefs(deviceMountPath string) ([]string, error)
 }
 
+// ExpandableVolumePlugin is an extended interface of VolumePlugin and is used for volumes that can be
+// expanded
+type ExpandableVolumePlugin interface {
+	VolumePlugin
+	NewExpander(spec *Spec) (Expander, error)
```

Comment:
Any reason not to add the `ExpandVolumeDevice(...)` and `RequiresFSResize(...)` methods directly here instead of burying them inside `Expander`? A lot of the code in `volumes.go` is unnecessary and we should clean that up. But for new code let's not add unnecessary cruft to match the existing code.